ext/bin/locust-load-tests-aggregator

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3.6
+
+import csv, os.path, sys, pry
+
+class CsvReport:
+    def __init__(self, header, body, footer):
+        self.header = header
+        self.body = body
+        self.footer = footer
+
+
+    def merge(self, csv_report):
+        # Appends a new csv_report object to the current instance,
+        # grouping is done by the key for header, body and footer
+        # csv_report - CsvReport -
+        existing_run_count = 0
+
+        for name, value in self.body.items():
+            existing_run_count = len(self.body[name])
+            new_value = csv_report.body.get(name, [''])
+            self.body[name].append(new_value[0])
+
+            if new_value != ['']: del csv_report.body[name]
+
+        self.header['Name'].append(csv_report.header['Name'][0])
+
+        if len(csv_report.body) > 0:
+          csv_report.body = { name: prepend_runs(run_time, existing_run_count) for name, run_time in csv_report.body.items()}
+          self.body.update(csv_report.body)
+
+        self.footer['Aggregated'].append(csv_report.footer['Aggregated'][0])
+
+        return self
+
+    def to_list(self):
+        # Converts header, body and footer to a list of lists representing rows in a csv file
+        data = []
+        for name, values in self.header.items():
+            data.append([name] + values)
+
+        for name, values in self.body.items():
+            data.append([name] + values)
+
+        for name, values in self.footer.items():
+            data.append([name] + values)
+
+        return data
+
+
+def usage():
+    usage = """
```

Comment:
the usage variable is not needed, you can print the string directly

Reply:
Updated, thanks!